test/bq.py

```diff
@@ -30,11 +28,45 @@ def list_table(self, table_id, limit=10):
         q = self.client.query(f'SELECT * FROM `{table_id}` LIMIT {limit}')
         return list(q.result())
 
+    def create_table(self, table_id, schema):
+        table = bigquery.Table(table_id, schema=schema)
+        try:
+            table = self.client.create_table(table)
+            log.info(f'Created table {table.project}.{table.dataset_id}.{table.table_id}')
+        except Conflict:
+            log.warning(f'Table {table.project}.{table.dataset_id}.{table.table_id} already exists')
+
+    def create_test_table(self, table_id):
+        schema = [
+            bigquery.SchemaField('t', 'TIMESTAMP', mode='REQUIRED'),
+            bigquery.SchemaField('u', 'INTEGER', mode='REQUIRED'),
+            bigquery.SchemaField('d', 'INTEGER', mode='REQUIRED'),
+            bigquery.SchemaField('m', 'INTEGER', mode='REQUIRED')
+        ]
+        self.create_table(table_id, schema)
+
+    def log_test_results(self, test_name, status, timestamp, create=False):
+        table_id = f'platform-dev-178517.bdc.integration_tests_{test_name}'
+        if status == 'success':
+            field = 'u'
+        elif status in ('failure', 'error'):
+            field = 'd'
+        elif status == 'skip':
+            field = 'm'
```

Comment:
Hmmm... the `m` for maintenance should never be logged currently.  This should be removed and a comment left instead.

Skipped tests should simply not send a row to BigQuery at all, ideally.